xbmc/pvr/PVRManager.cpp

```diff
@@ -466,6 +477,22 @@ bool CPVRManager::SetWakeupCommand(void)
   return false;
 }
 
+void CPVRManager::OnSleep()
+{
+  if (!IsStarted())
```

Comment:
Here the test checks is the PVRManager has fully started, not addon. What do you think to check instead ?